Automattic/jetpack
# My Jetpack: Add scaffolding for a Jetpack AI interstitial

Kick-off for #30553

Introduces route, interstitial and product class for a Jetpack AI product

Available under wp-admin/admin.php?page=my-jetpack#/add-jetpack-ai

Stlil needs work.

<img width="1110" alt="image" src="https://github.com/Automattic/jetpack/assets/746152/0e73aec0-8384-429c-af89-12530b78974e">

## Proposed changes:

* Introduces product class file for Jetpack AI
* Adds route `#/add-jetpack-ai`

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
No. Or yes. It Doesn't add a new event, but it also another path for tracking the event

## Testing instructions:






* Go to `wp-admin/admin.php?page=my-jetpack#/add-jetpack-ai`
* Conirm you see a new interstitial
